eks_fargate/README.md

```diff
@@ -2,23 +2,325 @@
+
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+ name: "<APPLICATION_NAME>"
+ namespace: default
+spec:
+ replicas: 1
+ template:
+   metadata:
+     labels:
+       app: "<APPLICATION_NAME>"
+     name: "<POD_NAME>"
+   spec:
+     serviceAccountName: datadog-agent
+     containers:
+     - name: "<APPLICATION_NAME>"
+       image: "<APPLICATION_IMAGE>"
+     ## Running the Agent as a side-car
+     - image: datadog/agent
+       name: datadog-agent
+       env:
+       - name: DD_API_KEY
+         value: "<YOUR_DATADOG_API_KEY>"
+         ## Set DD_SITE to "datadoghq.eu" to send your
+         ## Agent data to the Datadog EU site
+       - name: DD_SITE
+         value: "datadoghq.com"
+       - name: DD_EKS_FARGATE
+         value: "true"
+       - name: DD_KUBERNETES_KUBELET_NODENAME
+         valueFrom:
+           fieldRef:
+             apiVersion: v1
+             fieldPath: spec.nodeName
+      resources:
+          requests:
+            memory: "256Mi"
+            cpu: "200m"
+          limits:
+            memory: "256Mi"
+            cpu: "200m"
+```
+
+**Note**: Don't forget to replace `<YOUR_DATADOG_API_KEY>` with the [Datadog API key from your organization][13].
+
+## Metrics Collection
+
+### Integration metrics
+​
+Use [Autodiscovery labels with your application container][1] to start collecting its metrics for the [supported Agent integrations][2].
+​
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+ name: "<APPLICATION_NAME>"
+ namespace: default
+spec:
+ replicas: 1
+ template:
+   metadata:
+     labels:
+       app: "<APPLICATION_NAME>"
+     name: "<POD_NAME>"
+     annotations:
+      ad.datadoghq.com/<CONTAINER_NAME>.check_names: '[<CHECK_NAME>]'
+      ad.datadoghq.com/<CONTAINER_IDENTIFIER>.init_configs: '[<INIT_CONFIG>]'
+      ad.datadoghq.com/<CONTAINER_IDENTIFIER>.instances: '[<INSTANCE_CONFIG>]'
+   spec:
+     serviceAccountName: datadog-agent
+     containers:
+     - name: "<APPLICATION_NAME>"
+       image: "<APPLICATION_IMAGE>"
+     ## Running the Agent as a side-car
+     - image: datadog/agent
+       name: datadog-agent
+       env:
+       - name: DD_API_KEY
+         value: "<YOUR_DATADOG_API_KEY>"
+         ## Set DD_SITE to "datadoghq.eu" to send your
+         ## Agent data to the Datadog EU site
+       - name: DD_SITE
+         value: "datadoghq.com"
+       - name: DD_EKS_FARGATE
+         value: "true"
+       - name: DD_KUBERNETES_KUBELET_NODENAME
+         valueFrom:
+           fieldRef:
+             apiVersion: v1
+             fieldPath: spec.nodeName
+      resources:
+          requests:
+            memory: "256Mi"
+            cpu: "200m"
+          limits:
+            memory: "256Mi"
+            cpu: "200m"
+```
+
+**Note**: Don't forget to replace `<YOUR_DATADOG_API_KEY>` with the [Datadog API key from your organization][3].
+
+[1]: /agent/autodiscovery/integrations
+[2]: /integrations/#cat-autodiscovery
+[3]: https://app.datadoghq.com/account/settings#api
```

Comment:
Are those links correct? We use absolute ones usually no?

Reply:
indeed you're right, those needs to be updated. I'm on it :)